packages/sanity/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "@next-auth/sanity-adapter",
+  "description": "Sanity Adapter for NextAuth",
+  "version": "1.2.1",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "repository": "https://github.com/nextauthjs/adapters",
+  "author": "Fedeya <elfedeminaya@gmail.com>",
+  "license": "MIT",
+  "private": false,
+  "keywords": [
+    "next-auth",
+    "oauth",
+    "sanity",
+    "next.js"
+  ],
+  "scripts": {
+    "setup": "yarn build",
+    "build": "tsc"
+  },
+  "files": [
+    "README.md",
+    "dist"
+  ],
+  "devDependencies": {
+    "@types/lru-cache": "^5.1.0",
+    "@types/next-auth": "^3.15.0",
```

Comment:
```suggestion
```

This is an outdated type definition, `next-auth` ships with its own types now, so not needed!